etc/inc/notices.inc

```diff
@@ -350,6 +350,10 @@ function send_smtp_message($message, $subject = "(no subject)", $force = false)
 
 	if ($config['notifications']['smtp']['fromaddress']) {
 		$from = $config['notifications']['smtp']['fromaddress'];
+		preg_match('/[\w\s]+/', $from, $matches);
+		$from_display = trim($matches[0]);
+		if(preg_match('/(?:<)(.+)(?:>)$/', $from, $matches))
+			$from = $matches[1];
```

Comment:
Sorry, I don't know if I got it right. This change allow users to input "
renato@pfsense.org" or "Renato Botelho <renato@pfsense.org>", either way
will work.

You think it's better to have another field for $from_display?